ihatemoney/run.py

```diff
@@ -2,7 +2,7 @@
 import os.path
 import warnings
 
-from flask import Flask, g, request, session
+from flask import Flask, g, request, session, url_for
```

Comment:
Url for what ? :D

Reply:
Heh, it's coming from when I first tried to add the route in the run.py file (rather than in the web module)